pep/app/pods/components/page/drawer/template.hbs

```diff
@@ -112,30 +112,30 @@
                                 {{t "nav.user.accountInfo"}}
                             </button>
                         </Page::Drawer::NavItem>
-                    {{/if}}
-                    {{#if this.session.canLogOut}}
-                        {{#if this.session.isAuthenticated}}
-                            <Page::Drawer::NavItem class="mt-1">
-                                <button
-                                    type="button"
-                                    class="list-group-item list-group-item-action"
-                                    {{on "click" this.logout}}
-                                >
-                                    <FaIcon @icon="sign-out" @fixedWidth={{true}} class="mr-2" />
-                                    {{t "nav.user.logout"}}
-                                </button>
-                            </Page::Drawer::NavItem>
-                        {{else}}
-                            <Page::Drawer::NavItem class="mt-1">
-                                <button
-                                    type="button"
-                                    class="list-group-item list-group-item-action"
-                                    {{on "click" this.openLoginModal}}
-                                >
-                                    <FaIcon @icon="sign-in" @fixedWidth={{true}} class="mr-2" />
-                                    {{t "nav.user.login"}}
-                                </button>
-                            </Page::Drawer::NavItem>
+                        {{#if this.session.canLogOut}}
+                            {{#if this.session.isAuthenticated}}
```

Comment:
Im not sure the `else` here is reachable due to this all being inside of an `if` already that checks if your authenticated